notebooks/text_models/solutions/load_text.ipynb

```diff
@@ -0,0 +1,1935 @@
+{
```

Comment:
<p>It would be helpful to mention that <code>tf.keras.utils.text_dataset_from_directory</code> uses directory names as label names automatically, and it return tuples of <code>(feature, label)</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>+1</p>

Reply:
<p>Updated the markdown. Thank you! </p>